guacamole-docker/README.md

```diff
@@ -115,6 +119,35 @@ The process for doing this via the `mysql` utility included with MySQL is
 documented in
 [the Guacamole manual](http://guacamole.incubator.apache.org/doc/gug/jdbc-auth.html#jdbc-auth-mysql).
 
+
+Deploying Aoache Guacamole with LDAP authentication
+--------------------------------------------------
+
+    docker run --name some-guacamole --link some-guacd:guacd    \
+        -e LDAP_HOSTNAME=172.17.42.1                            \
+        -e LDAP_USER_BASE_DN=ou=people,dc=example,dc=com        \
+        -e LDAP_CONFIG_BASE_DN=ou=connections,dc=example,dc=com \
+        -d -p 8080:8080 glyptodon/guacamole
+
+Linking Guacamole to your LDAP directory will require additional configuration parameters
+specified via environment variables. These variables collectively describe how Guacamole
+will connect to LDAP:
+
+1. `LDAP_HOSTNAME` - The base of the DN for all Guacamole users.
+2. `LDAP_USER_BASE_DN` - The base of the DN for all Guacamole users.
+3. `LDAP_PORT` - The port your LDAP server listens on.  (Optional)
+4. `LDAP_ENCRYPTION_METHOD` - The encryption mechanism that Guacamole should use when
+communicating with your LDAP server. (Optional)
+5. `LDAP_GROUP_BASE_DN` - (Optional) The base of the DN for all groups that may be
```

Comment:
Why does this one start with "(Optional)" while all the others have "(Optional)" at the end?